### DISCUSSION TITLE
osu!catch scores imported from stable should not have a 0.96x score reduction

### DISCUSSION THREAD

**Discussion by Digitalfear117** [ppy/osu#26899] [Ideas]
0.96x score reduction may seem sensible for osu!standard due to the slider mechanics change, but there’s no major gameplay changes between stable and lazer for catch the beat. Since there is nothing that makes setting scores easier in stable, I feel this change should be reverted.

**Comment by Flawnpiece** [comment]
Since most of the points discussed in https://github.com/ppy/osu/discussions/26145 aren't going to be addressed soon, scoring should stay the same until some gameplay changes are introduced.

**Reply by Digitalfear117** [reply]
The thing about the ideas being proposed in that discussion, is that they would actually make lazer catch easier as most are bug fixes, or gameplay changes that produce more consistency. Take HR randomization removal, this would create less pixels and random hypers on jumps that otherwise wouldn’t have them.

**Reply by Flawnpiece** [reply]
Perhaps I wasn't clear but I completely agree with removing the current 0.96x score multiplier on catch. My point is that since none of the ideas (gameplay wise) proposed in that discussion weren't applied, catch gameplay is pretty much the same as in stable which make the 0.96x scoring useless. Once some of those points start to be implemented then we can have a discussion on multipliers and what not.

**Reply by Digitalfear117** [reply]
I do understand that, but what suggested gameplay change should even result in all (or some) stable scores getting a score reduction? Take for example the removal of HR randomization, or removal of droplet randomization, those things would actually make lazer catch easier. The reason for stable scores being reduction in standard is because lazer standard is harder (than stable).

**Reply by Flawnpiece** [reply]
I'm not saying that any of the suggested gameplay change should result in all stable scores getting a 0.96x score reduction. The point I'm making is that since stable and lazer gameplay are pretty much the same, there shouldn't be any score reduction. 

The reason why I brought up the suggested gameplay change is that once they happen we can start discussing of multipliers. If the removal of HR randomization makes lazer catch easier, then perhaps HR on lazer should have a score reduction of 0.95% compared to HR score from stable and so on.

**Reply by Digitalfear117** [reply]
Just wanted to make sure we were fully on the same page, thanks for clarifying 👍. I totally agree with what you are saying
